### PR TITLE
Logistics cleanup

### PR DESCRIPTION
**Pull request by kkrampa**

### CODE REVIEW

**Comment by czue** [comment]
```
Traceback (most recent call last):
  File "manage.py", line 88, in <module>
    execute_from_command_line(sys.argv)
  File "/vendor/lib/python2.7/site-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "/vendor/lib/python2.7/site-packages/django/core/management/__init__.py", line 377, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/vendor/lib/python2.7/site-packages/django/core/management/commands/test.py", line 50, in run_from_argv
    super(Command, self).run_from_argv(argv)
  File "/vendor/lib/python2.7/site-packages/django/core/management/base.py", line 288, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "/vendor/lib/python2.7/site-packages/django/core/management/commands/test.py", line 71, in execute
    super(Command, self).execute(*args, **options)
  File "/vendor/lib/python2.7/site-packages/django/core/management/base.py", line 338, in execute
    output = self.handle(*args, **options)
  File "/vendor/lib/python2.7/site-packages/django/core/management/commands/test.py", line 88, in handle
    failures = test_runner.run_tests(test_labels)
  File "/mnt/testrunner.py", line 385, in run_tests
    all_suite = self.build_suite(all_test_labels, extra_tests)
  File "/vendor/lib/python2.7/site-packages/django/test/simple.py", line 242, in build_suite
    suite.addTest(build_suite(app_config))
  File "/vendor/lib/python2.7/site-packages/django/test/simple.py", line 144, in build_suite
    tests_module = get_tests(app_config)
  File "/vendor/lib/python2.7/site-packages/django/test/simple.py", line 102, in get_tests
    test_module = import_module('%s.%s' % (app_config.name, TEST_MODULE))
  File "/usr/local/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/mnt/custom/logistics/tests/__init__.py", line 2, in <module>
    from .test_delete_domain import *
ImportError: No module named test_delete_domain
```

**Comment by kkrampa** [comment]
:white_check_mark:

**Comment by czue** [comment]
lgtm. will let @gcapalbo review

File: custom/ilsgateway/migrations/0008_auto_20160401_0807.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    operations = [
+        migrations.AlterUniqueTogether(
+            name='historicallocationgroup',
+            unique_together=None,
+        ),
+        migrations.RemoveField(
+            model_name='historicallocationgroup',
+            name='location_id',
+        ),
```
Comment (gcapalbo) [inline custom/ilsgateway/migrations/0008_auto_20160401_0807.py]: no need to change, but am just curious why you had to drop this field first before dropping the whole model

File: custom/ewsghana/models.py [hunk 2]
```diff
@@ -173,7 +132,5 @@ def domain_pre_delete_receiver(domain, **kwargs):
     from corehq.apps.domain.deletion import ModelDeletion
     return [
         ModelDeletion('ewsghana', 'FacilityInCharge', 'location__domain'),
-        ModelDeletion('ewsghana', 'EWSExtension', 'domain'),
```
Comment (gcapalbo) [inline custom/ewsghana/models.py]: Did you mean to remove EWSExtension?

File: custom/ilsgateway/migrations/0008_auto_20160401_0807.py (hunk 1 above)
Comment (kkrampa) [inline custom/ilsgateway/migrations/0008_auto_20160401_0807.py]: I have no idea. It's automatically generated by django.

File: custom/ewsghana/models.py (hunk 2 above)
Comment (kkrampa) [inline custom/ewsghana/models.py]: @gcapalbo It's not removed. I only removed comma: https://github.com/dimagi/commcare-hq/pull/11042/files#diff-251f5e8a24177a37339ee2edcd513efbR135

Comment (gcapalbo) [inline custom/ewsghana/models.py]: (sorry i looked too fast)

**Pull request merged**